resteasy-client/src/main/java/org/jboss/resteasy/plugins/providers/sse/client/SseEventSourceImpl.java

```diff
@@ -304,10 +306,11 @@ public void run()
          long delay = reconnectDelay;
          try
          {
-            final Invocation.Builder request = buildRequest();
+            final  ClientInvocation invocation = (ClientInvocation) buildRequest().buildGet();
             if (state.get() == State.OPEN)
             {
-               eventInput = request.get(SseEventInputImpl.class);
+               response = (ClientResponse)invocation.invoke();
+               eventInput = ClientInvocation.extractResult(new GenericType<SseEventInputImpl>(SseEventInputImpl.class), response, null);
```

Comment:
If current SseEventSource has been closed after getting the response but before reading the SseEventInputImpl, invoking ClientInvocation.extractResult(new GenericType<SseEventInputImpl>(SseEventInputImpl.class), response, null) will throw an ResponseProcessingException with IllegalStateException as root cause. This exception is not an error but a normal consequence of the SseEventSource being closed by user so let's just ignore it instead of sending to error consumers. WDYT ?

Reply:
I am not sure if this will happen . If there is exception, I think it doesn't matter to trigger the error consumers.

Reply:
OK